ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/composables/Markdown.kt

```diff
@@ -388,10 +540,14 @@ private fun MarkdownText(
         text = text,
         color = color,
         style = style,
+        fontSize = fontSize,
         fontWeight = fontWeight,
+        fontFamily = fontFamily,
         textAlign = textAlign,
         modifier = modifier
-            .fillMaxWidth()
+            .conditional(textFillMaxWidth) {
+                fillMaxWidth()
+            }
```

Comment:
This is what brought me a headache in this change... Tried to minimize the visual changes with this.